lib/html.js

```diff
@@ -61,7 +61,12 @@ helpers.css = function(list, options) {
 
   return styles.map(function(item) {
     var ext = path.extname(item);
-    var fp = path.posix.join(assets, item);
+    var fp = item;
+
+    var isExternal =  /(^\/\/)|(:\/\/)/.test(item);
```

Comment:
lint error: There are multiple spaces before the regexp

Reply:
This can be inlined inside the `if` conditional expression since `isExternal` isn't used again.